### DISCUSSION TITLE
longer sampling .wavs?

### DISCUSSION THREAD

**Discussion by ghost** [neonbjb/tortoise-tts#236] [General]
I accidentally forgot to cut a 2m audio file into 10-15s, and it doesn't seem to have hurt the quality of output at all. Just thought I'd bring this discovery. Not sure if it helped or hurt or did anything in particular, lol

**Comment by ef1500** [comment]
I don't think that it should make much difference. I've been toying around with it for a while and I've noticed that the samples seem to fluctuate from anywhere between 3 seconds and upwards of around 20 seconds. I think you should be fine. Play around, experiment, find what works best for you.

**Reply by RichieRunner** [reply]
Have you been able to draw any conclusions on what seems to be a suitable combination of audio sample length and number of audio samples?

**Reply by ef1500** [reply]
### **TL;DR** It depends on how much of the sample actually contains audio.

I posted in another [discussion ](https://github.com/neonbjb/tortoise-tts/discussions/267#discussioncomment-4910536) that it took me 132 samples to get the female voice to not come out British. 

I noted in that discussion that I had to delete the ones that didn't have any audio in them or had very little. After repeating this process for a while, I trimmed my 132 Samples to about 23. This seemed to work fine as well. 

My current set contains only 23 voice samples, each approximately 10 seconds long, this seems to provide a stable output. It should be noted that each of these 23 samples has audio playing throughout most of the audio samples.

I accidentally figured out that MP3 Files seem to work too, but not very well. I'd stick to the wav format.

As a general rule of thumb, remember that the output is going to sound like the "average" of all of the audio files combined, so **garbage in, garbage out**. 

Hope this helps ya out, cheers :)

**Reply by FurkanGozukara** [reply]
> ### **TL;DR** It depends on how much of the sample actually contains audio.
> I posted in another [discussion ](https://github.com/neonbjb/tortoise-tts/discussions/267#discussioncomment-4910536) that it took me 132 samples to get the female voice to not come out British.
> 
> I noted in that discussion that I had to delete the ones that didn't have any audio in them or had very little. After repeating this process for a while, I trimmed my 132 Samples to about 23. This seemed to work fine as well.
> 
> My current set contains only 23 voice samples, each approximately 10 seconds long, this seems to provide a stable output. It should be noted that each of these 23 samples has audio playing throughout most of the audio samples.
> 
> I accidentally figured out that MP3 Files seem to work too, but not very well. I'd stick to the wav format.
> 
> As a general rule of thumb, remember that the output is going to sound like the "average" of all of the audio files combined, so **garbage in, garbage out**.
> 
> Hope this helps ya out, cheers :)

do you have any automated way to generate this 10 second clips?

**Comment by FurkanGozukara** [comment]
@ef1500 do you prefer 32fp 22050 or 16fp 22050 
which one working better you have found?
